### DISCUSSION TITLE
Support for gtk4

### DISCUSSION THREAD

**Discussion by gdimartino** [jralls/gtk-osx-build#79] [General]
I'm brand new to GTK and have just gotten my first gtk3 hello world program to compile and run using gtk-osx-build for the setup!

I am wondering if there is any plan to support gtk4 in the near future?

Thank you!

**Comment by jralls** [comment]
[It's already supported.](https://github.com/jralls/gtk-osx-build/blob/625df3e90edacc2472640dc7f4d937b9e823fbbb/modulesets-stable/gtk-osx.modules#L378)
